### ISSUE TITLE
Offensive language

### ISSUE THREAD

**Issue by yoursdearboy** [perwendel/perwendel.github.io#14]
Greetings.
I stumbled upon the Spark website recently and noticed this paragraph in section for Node.js developers:

> You also get all the benefits of running your application on the JVM, where libraries aren't deprecated every day.

Sounds a bit offensive (and I'm not a JS developer). That's a good point, but don't you think to rephrase it?

**Comment by tipsy** [comment]
I agree that it's unnecessarily hostile. Feel free to submit a PR with better wording, and I'll merge it.
